plasmapy/physics/parameters.py

```diff
@@ -790,24 +798,19 @@ def plasma_frequency(n, particle='e'):
     >>> from astropy import units as u
     >>> plasma_frequency(1e19*u.m**-3, particle='p')
     <Quantity 4163294530.6925354 rad / s>
-    >>> plasma_frequency(1e19*u.m**-3, particle='p')
-    <Quantity 4163294530.6925354 rad / s>
+    >>> plasma_frequency(1e19*u.m**-3, particle='D+')
+    <Quantity 2944624520.2526317 rad / s>
     >>> plasma_frequency(1e19*u.m**-3)
     <Quantity 178398636471.3789 rad / s>
-
     """
 
     try:
         m = atomic.ion_mass(particle)
-        try:
-            Z = atomic.charge_state(particle)
-        except ValueError:
-            Z = 1
     except Exception:
-        raise ValueError(f"Invalid particle, {particle}, in "
+        raise ValueError("Invalid particle, {particle}, in "
```

Comment:
Keep this as an f string or it will literally print {particle}!

Reply:
Indeed ! I didn't knew about this convenient format shorcut, nice !